packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -90,9 +92,27 @@ export class CommandRegistryImpl implements CommandRegistryExt {
     executeCommand<T>(id: string, ...args: any[]): PromiseLike<T | undefined> {
         if (this.handlers.has(id)) {
             return this.executeLocalCommand(id, ...args);
-        } else {
+        } else if (KnownCommands.exclusions.hasOwnProperty(id)) {
```

Comment:
If I understand correctly, only when an incoming `id` is in the `exclusion` map do we call `KnownCommands.map`, which because of this condition will branch in the first `exclusions[id]` check. Then my question is: When will it go in the `mappings[id]` branch?